utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/encode/ValidationMode.scala

```diff
@@ -1,14 +1,14 @@
 package pl.touk.nussknacker.engine.avro.encode
 
-case class ValidationMode private(name: String, label: String, acceptRedundant: Boolean)
+case class ValidationMode private(name: String, label: String, strict: Boolean)
```

Comment:
We we still have this strict field if you can just check if `mode == ValidationMode.strict`?

Reply:
Hmm, I've removed field strinct and right now operate on const: `strict` as you said.